docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -330,6 +327,45 @@ On touch devices, TestCafe emulates touch events instead of mouse events.
 
 Mouse event | Touch event
 ----------- | -------------
-`mousemove` (when hovering or dragging) | `touchmove` (dragging only)
+`mousemove` (when hovering or dragging) | `touchmove` (when dragging)
 `mousedown` | `touchstart`
 `mouseup`   | `touchend`
+
+## Interaction Requirements
+
+TestCafe actions can interact with elements if they satisfy the following conditions:
+
+* an element is within the page's [`body`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body) or in an [`<iframe>`](#work-with-iframes). The element can be invisible for the user. If the element is off-screen, TestCafe attempts to reach it with a scroll.
+
+* an element is visible - its properties are as follows:
+
+    Property | Value
+    -------- | --------
+    `display`  | *not* set to `none`
+    `visibility` | set to `visible` (the default value)
+    `width`    | auto calculated as or set to >= 1px
+    `height`   | auto calculated as or set to >= 1px
+
+* an element isn't obstructed or overlapped.  
+
+    TestCafe actions target a center of an element, or a point specified by the `offsetX` and `offsetY` options of an action. If another element obstructs the target point, the action is executed with the overlapping element (for instance, the [t.click](../../reference/test-api/testcontroller/click.md) action clicks an overlapping block).
+
+### Example: Scroll an Element into View
+
+TestCafe scrolls to reach items that are on the page but not on-screen. Therefore, TestCafe API doesn't have a dedicated scroll action.
```

Comment:
I'm not sure why you used "Therefore". Did you mean to say "However"? For example:

`TestCafe can scroll to reach items that are on the page but not on-screen. However, the TestCafe API does not have a dedicated scroll action.`

Reply:
This is an attempt to underline causality.

All TestCafe actions incorporate scroll into their routine. **For that reason**, there is no designated scroll action in TestCafe as there is no need for it.

Reply:
Thanks for the clarification. How about:

`Since TestCafe scrolls to reach items that are on the page but not on-screen, the TestCafe API does not have a dedicated scroll action.`

Reply:
Good one, edited